src/ansys/dpf/gate/load_api.py

```diff
@@ -51,38 +51,51 @@ def _pythonize_awp_version(version):
     return "20" + version[0:2] + "." + version[2]
 
 
-def _find_latest_ansys_versions():
-    awp_versions = [key[-3:] for key in os.environ.keys() if "AWP_ROOT" in key]
-    installed_packages_list = {}
+def _find_latest_dpf_server():
+    path_per_version = {}
 
-    for awp_version in awp_versions:
-        if not awp_version.isnumeric():
-            continue
-        ansys_path = os.environ.get("AWP_ROOT" + awp_version)
-        if ansys_path:
-            installed_packages_list[
-                packaging.version.parse(_pythonize_awp_version(awp_version))
-            ] = ansys_path
+    path_per_version = _paths_to_dpf_in_unified_installs(path_per_version)
+
+    path_per_version = _paths_to_dpf_server_library_installs(path_per_version)
```

Comment:
@PProfizi As we are adding entries to the same dictionary (`path_per_version`), do we risk overwritten already existing entries? Or this is the expected behavior as per your proposal?

Reply:
@rafacanton `AWP_ROOTXXX` variables give 'production' versions (such as `2024.2`), see `_pythonize_awp_version`, while `ansys_dpf_server` versions are always pre-production (such as `2024.2.pre0`) so IMO they should not intersect...